packages/ivh-base-style/demo.html

```diff
@@ -594,30 +594,178 @@ <h2>Tables</h2>
           <p>For tabular data only!</p>
         </div>
 
-        <div class="demo">
-          <h2>A True Table</h2>
+
+      <div class="demo">
```

Comment:
Just a little style nitpick but I believe this section is missing one level of indentation.